draft-ietf-taps-transport-security.md

```diff
@@ -159,7 +160,9 @@ efforts to define and catalog transport services {{RFC8095}} by describing the i
 add security protocols. It examines Transport Layer Security (TLS), Datagram Transport Layer 
 Security (DTLS), Quick UDP Internet Connections with TLS (QUIC + TLS), MinimalT, CurveCP, tcpcrypt, 
 Internet Key Exchange with Encapsulating Security Protocol (IKEv2 + ESP), SRTP (with DTLS), and 
-WireGuard. This survey is not limited to protocols developed within the scope or context of the IETF.
+WireGuard. Selected protocols represent a superset of functionality and features a TAPS system may 
+need to support, both internally and externally -- via an API -- for applications. As such, 
+this survey is not limited to protocols developed within the scope or context of the IETF.
```

Comment:
I think we can expand this a bit, perhaps a new paragraph. The discussion at the meeting was about telling the story about how we selected each, a bit.